src/containers/cooperation-details/cooperation-from-scratch/CooperationFromScratch.styles.ts

```diff
@@ -35,5 +35,12 @@ export const styles = {
   resourcesSelect: {
     width: { sm: '200px', xs: '120px' },
     cursor: 'pointer'
+  },
+  buttons: {
+    display: 'flex',
+    gap: { xs: '24px', sm: '30px' },
+    justifyContent: 'end',
+    alignSelf: { xs: 'center', sm: 'end' },
+    marginTop: { xs: '5px', sm: '20px' }
```

Comment:
mt here

Reply:
marginTop also could be

Reply:
> marginTop also could be

we stick to shorter convention, should rewrite

Reply:
done